applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -161,6 +161,20 @@ static struct {
 	} pres_comp;
 } ctrl_blk;
 
+/**
+ * @brief	Get the current number of blocks in the output buffer.
+ */
+static int used_blocks_get(void)
```

Comment:
Nit: 'Used' sounds like they are spent

```suggestion
static int filled_blocks_get(void)
```

Reply:
Agreed, changed.

Reply:
Did you forget to add before committing and pushing? There are no changes in the last commit